cdap-app-fabric/src/test/java/co/cask/cdap/metadata/LineageAdminTest.java

```diff
@@ -138,20 +157,21 @@ public void testSimpleLoopLineage() throws Exception {
     LineageStore lineageStore = new LineageStore(getTxExecFactory(), getDatasetFramework(),
                                                  Id.DatasetInstance.from("default", "testSimpleLoopLineage"));
     Store store = getInjector().getInstance(Store.class);
-    LineageGenerator lineageGenerator = new LineageGenerator(lineageStore, store);
+    BusinessMetadataStore businessMetadataStore = getInjector().getInstance(BusinessMetadataStore.class);
+    LineageAdmin lineageAdmin = new LineageAdmin(lineageStore, store, businessMetadataStore);
 
 
     // Add access
     addRuns(store, run1, run2, run3, run4, run5);
-    lineageStore.addAccess(run1, dataset1, AccessType.READ, EMPTY_METADATA, flowlet1);
-    lineageStore.addAccess(run1, dataset2, AccessType.WRITE, EMPTY_METADATA, flowlet1);
+    lineageStore.addAccess(run1, dataset1, AccessType.READ, System.currentTimeMillis(), flowlet1);
```

Comment:
can you add a comment on why using ``System.currentTimeMillis`` is ok in this case (as opposed to using a known long value)?